app/uk/gov/hmrc/vatregisteredcompanies/test/TestController.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatregisteredcompanies.test
+
+import javax.inject.Inject
+import play.api.mvc.{Action, AnyContent}
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+
+import scala.concurrent.Future
+
+class TestController @Inject()(
+  connector: TestConnector
+) extends FrontendController {
+
+  def triggerDataImport: Action[AnyContent] = Action.async { implicit request =>
+    connector.trigger("trigger-mdg-data-post") flatMap (_ =>
```

Comment:
`a >> b` is the same as `a.flatMap{ _ => b}`. It's a bit clearer imho but might confuse people who have not come across it before.